### DISCUSSION TITLE
Button text colour

### DISCUSSION THREAD

**Discussion by avalon60** [Akascape/CTkTable#21] [Ideas]
I have a suggestion. Presently you pick up the default text color from CTkButton. However, there is less chance of this contrasting well with the button colours being rendered, since these are based on CTkFrame colours. Instead it would be better, to grab the text_color from CTkLabel.

Here is how my theme looks:

![image](https://github.com/Akascape/CTkTable/assets/89534395/9bd47cec-351a-4f7a-8fbb-2db11f5a1c06)

Here is an example of how CTkTable can get it wrong:

![image](https://github.com/Akascape/CTkTable/assets/89534395/34d5954f-d0ff-4c10-a52f-951098572df7)

However if you use CTkLabel, you get this:

![image](https://github.com/Akascape/CTkTable/assets/89534395/2bca7cd9-809a-4d7e-b7b7-6d8e4c1fe772)

Label text colours tend to be chosen so that they work well with the Frame colours.

**Comment by Akascape** [comment]
@avalon60 Done. changed from button text to label text color.
